processor/k8sattributesprocessor/README.md

```diff
@@ -82,6 +105,22 @@ Additional container level attributes can be extracted provided that certain res
    instance. If it's not set, the latest container instance will be used:
    - container.id (not added by default, has to be specified in `metadata`)
 
+Please note however that container level attributes can not be used for source rules in the pod_association.
+
+Example for extracting container level attributes:
+
+```yaml
+extract:
```

Comment:
While this example is useful, I think we lack an example showing what the output of the processor looks like given specific inputs.

Reply:
you're right - having an example here should make things clearer - I have added a small example to illustrate how the attributes of a span are extended based on the properties of a matching pod